app/Filament/Resources/DocumentResource/RelationManagers/RevisionsRelationManager.php

```diff
@@ -0,0 +1,87 @@
+<?php
+
+namespace App\Filament\Resources\DocumentResource\RelationManagers;
+
+use App\Filament\Resources\RevisionResource;
+use App\Models\Document;
+use App\Models\Revision;
+use Filament\Forms;
+use Filament\Forms\Form;
+use Filament\Resources\RelationManagers\RelationManager;
+use Filament\Tables;
+use Filament\Tables\Table;
+use Illuminate\Support\Carbon;
+use Livewire\Features\SupportFileUploads\TemporaryUploadedFile;
+
+class RevisionsRelationManager extends RelationManager
+{
+    protected static string $relationship = 'revisions';
+
+    public function form(Form $form): Form
+    {
+        return $form
+            ->schema([
+                Forms\Components\DatePicker::make('date')
+                    ->label(__('Revision date'))
+                    ->format('Y-m-d')
+                    ->unique()
+                    ->default(Carbon::now())
+                    ->validationMessages([
+                        'unique' => __('A revision with this date already exists.'),
+                    ])
+                    ->columnSpan(2),
+                Forms\Components\FileUpload::make('file.en')
+                    ->label(__('File (English)'))
+                    ->requiredWithout('file.fr')
+                    ->disk('public')
+                    ->directory('documents')
+                    ->getUploadedFileNameForStorageUsing(function (?Revision $record, TemporaryUploadedFile $file, RelationManager $livewire): string {
+                        /** @var Document */
+                        $document = $livewire->getOwnerRecord();
+
+                        return RevisionResource::getFilename('en', $file->extension(), $document, $record);
+                    }),
+                Forms\Components\FileUpload::make('file.fr')
+                    ->label(__('File (French)'))
+                    ->requiredWithout('file.en')
+                    ->disk('public')
+                    ->directory('documents')
+                    ->getUploadedFileNameForStorageUsing(function (?Revision $record, TemporaryUploadedFile $file, RelationManager $livewire): string {
+                        /** @var Document */
+                        $document = $livewire->getOwnerRecord();
+
+                        return RevisionResource::getFilename('fr', $file->extension(), $document, $record);
+                    }),
+            ]);
+    }
+
+    public function table(Table $table): Table
+    {
+        return $table
+            ->recordTitleAttribute('create_at')
+            ->columns([
+                Tables\Columns\TextColumn::make('created_at')->label(__('Revision date'))
```

Comment:
The revisions table is showing the created date instead of the revision date
```suggestion
                Tables\Columns\TextColumn::make('date')->label(__('Revision date'))
```